requirements/base.txt

```diff
@@ -20,8 +20,8 @@ django-debug-toolbar==3.2.1
 newspaper3k==0.2.8
 
 git+https://github.com/DemocracyClub/dc_signup_form.git@2.1.0
-git+https://github.com/DemocracyClub/design-system.git@0.1.5
-git+https://github.com/DemocracyClub/dc_django_utils.git@0.0.5
+git+https://github.com/DemocracyClub/design-system.git@0.1.4
```

Comment:
Is this version drop to 0.1.4 intended?

Reply:
No, but I didn't think it should go here.

Reply:
[The increase to 0.1.5 already got merged to master ](https://github.com/DemocracyClub/WhoCanIVoteFor/blob/master/requirements/base.txt#L23) so this is actually a change down a version, which would bring back the logo error. Changing back to 0.1.5 here will stop the line showing up as a change.